train_management/templates/train_management/reservation_calendar.html

```diff
@@ -0,0 +1,197 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+
+{% block title %}{% translate "reservation_calendar.title" %}{% endblock %}
+{% block header_scripts %}
+<script src="https://unpkg.com/js-year-calendar@latest/dist/js-year-calendar.min.js"></script>
+
+<link rel="stylesheet" type="text/css" href="https://unpkg.com/js-year-calendar@latest/dist/js-year-calendar.min.css"/>
+<script src="https://unpkg.com/js-year-calendar@latest/locales/js-year-calendar.de.js"></script>
+{% endblock header_scripts %}
+{% block content %}
+
+
+<div class="dvzo-header">
+  {% translate "reservation_calendar.title" %}
+</div>
+
+<div class="dvzo-content">
+  <div class="row">
+    <div class="col">
+      <div class="card my-2">
+        <div class="card-body">
+          <div class="card-text">
+            <div id="calendar"></div>
+
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+<!--
+ TODO:
+ - change dataSource to be served by django
+ -- each dayplanning is an dataSource entry
+ --- id = Dayplanning.id
+ --- name = Dayplanning.label
+ --- url = described later
+ --- startDate = Dayplanning.date
+ --- endDate = Dayplanning.date
+ --- color = DayplanningType
+      SUNDAY = '#ed174b'
+      EXTRA = '#6a99e6'
+      OTHER = 'grey'
+ - The calendar will be used for two things:
+ -- see the dayplannings in a calendar view. reached from dayplanning-list.
+ --- url = /dayplanning/detail/<id>
+ -- make reservations:
+ --- url = /reservation-calendar/<dayplanning.id>
+
+ -->
+<script>
+  new Calendar('#calendar', {
```

Comment:
Solange wir nicht rest verwenden würde ich schauen, ob wir die Daten nicht auch in HTML rendern können.